### DISCUSSION TITLE
Unable to add projects to the community exchange

### DISCUSSION THREAD

**Discussion by clicktodev** [community/community#144721] [GitHub Education]
### Hi everyone,

Attempting to add a repo with Learn and Collaborate selected is broken at https://education.github.com/globalcampus/exchange

after pressing submit, the following error shows up

![image](https://github.com/user-attachments/assets/7b8d911c-8c95-432a-9b68-77e108071e88)

**Comment by BancoCentraI** [comment]
Potential Causes and Solutions
Here are a few common reasons and troubleshooting steps you can try:

Clear Browser Cache and Cookies:

Sometimes, stored cache or cookies can interfere with site functionality. Clearing them and restarting the browser might resolve the issue.
Use Incognito Mode:

If clearing the cache doesn’t work, try using your browser’s incognito/private mode to rule out any extensions or cached data conflicts.
Check for JavaScript Errors:

If you’re comfortable with browser developer tools, press F12 (or right-click -> Inspect) to open the console. Check for any red error messages when you try to submit the form. This might provide a clue about what's breaking.
Try a Different Browser or Device:

Sometimes, compatibility issues can arise with specific browsers. Try switching to a different one (e.g., Chrome, Firefox, Safari) to see if the issue persists.
GitHub Status Page:

GitHub services sometimes experience outages or maintenance. Check [GitHub Status](https://www.githubstatus.com/) to see if there are any ongoing issues affecting Global Campus.
Contact GitHub Support:

If none of the above resolves your issue, it might be a problem specific to your account or a bug with the Global Campus platform. Report it to [GitHub Education Support](https://support.github.com/contact/education) with details of the error message and your steps.
